app/classes/labels.rb

```diff
@@ -119,7 +118,7 @@ def add_name
     italic = false
     @para.bold do |bold|
       @obs.name.display_name.gsub("**", "").split("__").each do |part|
-        if part.present?
+        unless part.empty?
           if italic
             bold.italic { |i| i << part } if part.present?
           else
```

Comment:
Seems like we're reinventing the wheel here.
Are we sure we don't have a Textile or String class method for this already?